src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.vue

```diff
@@ -28,11 +28,20 @@
       class="optionsRow"
     >
       <ft-toggle-switch
+        class="matchingVideoToggle"
         :label="$t('User Playlists.Playlists with Matching Videos')"
         :compact="true"
         :default-value="doSearchPlaylistsWithMatchingVideos"
         @change="doSearchPlaylistsWithMatchingVideos = !doSearchPlaylistsWithMatchingVideos"
       />
+      <ft-toggle-switch
+        v-if="anyPlaylistContainsVideosToBeAdded"
+        class="allowDuplicateToggle"
+        :label="$t('User Playlists.AddVideoPrompt.Allow Adding Duplicate Video(s)')"
+        :compact="true"
+        :default-value="addingDuplicateVideosEnabled"
+        @change="addingDuplicateVideosEnabled = !addingDuplicateVideosEnabled"
+      />
```

Comment:
**suggestion (blocking):** Thanks for moving these to the right. Could you also stack these two toggles vertically (to increase alignment and further visually group the controls)?

Reply:
I don't think we have any existing design for having vertical grouping inside a row unless width is passed a breakpoint

![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/0f590a1b-c3f6-4747-af08-05d8aaf23d71)

This is not like settings which we have so many so that they are aligned horizontally with limited slots per row (well a table)
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/5fde09b0-69fb-4cc1-9dae-b90990f1b477)

Reply:
I'm having trouble understanding, are you expressing that it's difficult to implement or shouldn't be implemented for X reason?

Reply:
> Could you also stack these two toggles vertically

I posted screenshot above (not pushed) to see (1) if this is what you want (2) if it is what you want then it shouldn't be like that since it's not a table and there is enough width

Reply:
I'm still not sure I understand. You can put the two toggles in a flex container that is `inline-end`-aligned, and then have that flex container's content be `inline-start`-aligned.

Reply:
Short version: it looks weird
Long version: It's strange to see 2 elements/components aligned vertically in a row which is not a table and IMO is not an existing design in FT (feel free to name other projects with this design or existing places in FT with this design in case I miss). At the same time these 2 components are only common in type (toggle) but not their purpose. One is for searching/filtering and another one is for affecting the **outcome** of the action. (1st reason is more important than 2nd one)

Reply:
I personally disagree, but this is not a blocking concern, so will leave it to others' discretion and not push the matter